ydb_sqlalchemy/dbapi/cursor.py

```diff
@@ -21,75 +18,22 @@
 logger = logging.getLogger(__name__)
 
 
-identifier_starts = {x for x in itertools.chain(string.ascii_letters, "_")}
-valid_identifier_chars = {x for x in itertools.chain(identifier_starts, string.digits)}
-
-
-def check_identifier_valid(idt: str) -> bool:
-    valid = idt and idt[0] in identifier_starts and all(c in valid_identifier_chars for c in idt)
-    if not valid:
-        raise ProgrammingError(f"Invalid identifier {idt}")
-    return valid
-
-
 def get_column_type(type_obj: Any) -> str:
     return str(ydb.convert.type_to_native(type_obj))
 
 
-def _generate_type_str(value: Any) -> str:
-    tvalue = type(value)
-
-    stype = {
-        bool: "Bool",
-        bytes: "String",
-        str: "Utf8",
-        int: "Int64",
-        float: "Double",
-        decimal.Decimal: "Decimal(22, 9)",
-        datetime.date: "Date",
-        datetime.datetime: "Timestamp",
-        datetime.timedelta: "Interval",
-        uuid.UUID: "Uuid",
-    }.get(tvalue)
-
-    if tvalue == dict:
-        types_lst = ", ".join(f"{k}: {_generate_type_str(v)}" for k, v in value.items())
-        stype = f"Struct<{types_lst}>"
-
-    elif tvalue == tuple:
-        types_lst = ", ".join(_generate_type_str(x) for x in value)
-        stype = f"Tuple<{types_lst}>"
-
-    elif tvalue == list:
-        nested_type = _generate_type_str(value[0])
-        stype = f"List<{nested_type}>"
-
-    elif tvalue == set:
-        nested_type = _generate_type_str(next(iter(value)))
-        stype = f"Set<{nested_type}>"
-
-    if stype is None:
-        raise ProgrammingError(f"Cannot translate value {value} (type {tvalue}) to ydb type.")
+class YdbOperation:
+    __slots__ = ("yql_text", "is_ddl", "parameters_types")
```

Comment:
What about:
use dataclasses with default values?

[Similar to ydb sdk](https://github.com/ydb-platform/ydb-python-sdk/blob/29e603e9440ea3c8b359a8dff8cd6c0aac82f3d4/ydb/topic.py#L388-L388)

Reply:
Done